core/src/main/java/com/linecorp/armeria/client/DefaultClientRequestContext.java

```diff
@@ -97,15 +98,16 @@
      *
      * @param eventLoop the {@link EventLoop} associated with this context
      * @param sessionProtocol the {@link SessionProtocol} of the invocation
+     * @param uuid the {@link UUID} that contains the information about the current {@link HttpRequest}.
```

Comment:
`uuid the {@link UUID} that represents the current {@link Request} and {@link Response}`

Reply:
`... represents the unique identifier of the current {@link Request} and {@link Response} pair.`